share/availability/test/testing.go

```diff
@@ -72,12 +72,9 @@ func NewTestDAGNet(ctx context.Context, t *testing.T) *DagNet {
 	}
 }
 
-// Node create a plain network node that can serve and request data.
-func (dn *DagNet) Node() *Node {
+func (dn *DagNet) NodeWithBlockStore(dstore ds.Datastore, bstore blockstore.Blockstore) *Node {
```

Comment:
Does this need to be exported? If not, I'd recommend non-exported test methods by default.

Reply:
It does need to be exported because it comes from a different package (and not a _test.go file, so exporting works)